src/components/Button/index.js

```diff
@@ -1,22 +1,49 @@
 import React, {PropTypes} from 'react'
 
-const Button = ({
-  children, 
-  onClick, 
-  className,
-}) => (
-  <button
-    className={`button-reset bn dib b0 ttu dim br2 ph3 pv2 white bg-orange tc ${className}`}
-    onClick={onClick}
-  >
-    {children}
-  </button>
-)
+const commonClasses = 'link dib f5 fw6 tracked tc br2 ttu ba pointer'
+
+const sizes = ['large', 'small', 'cta']
```

Comment:
`cta` as a size is 😢 

It's definitely not a size. It's a name a designer gave a particular style in a comp. If it is special it should probably be its own meta-Button and not included in the core framework IMO.